### ISSUE TITLE
Output

### ISSUE THREAD

**Issue by WoutVanEynde** [12rajnish/DeePred-BBB#1]
Dear

I had a question regarding the output:

Is it possible to change the code in a way that next to the outcome (BBB-nonpermeable or BBB-permeable), the original SMILES would appear? Because it seems that if we input a batch of molecules the input order gets rearranged and therefore batch analysis becomes unusable. 

Warm regards,
Wout Van Eynde

**Comment by 12rajnish** [comment]
DeePred-BBB does not take multiple SMILES notations for prediction. The user needs to input the SMILES notations of compounds one at a time for accurate prediction of BBB permeability.

This limitation is also mentioned in the published paper of DeePred-BBB (attached for reference)
[fnins-16-858126.pdf](https://github.com/12rajnish/DeePred-BBB/files/8926397/fnins-16-858126.pdf)

**Comment by WoutVanEynde** [comment]
Dear

I read it in the paper after I posted it, my excuses.

Thank you for your fast answer.

Warm regards,
Wout Van Eynde

**Comment by chushuang203** [comment]
Hello, I defined a DNN model based on the article and found that the training loss is NaN using part of the dataset from the supplementary materials. Would you do me a favor? Thanks!
